### ISSUE TITLE
Some issues

### ISSUE THREAD

**Issue by zhangxiaoshuotttt** [GeophyAI/seistorch#1]
1. Here now we must set as [these parameters](https://github.com/GeophyAI/seistorch/blob/202c77abbdc30917c3b36c383c28ded0472c35b2/template.yml#L123) and COULDN'T change them.

**Comment by zhangxiaoshuotttt** [comment]
2. This should be [` python ../../../fwi.py forward.yml \ ` ](https://github.com/GeophyAI/seistorch/blob/202c77abbdc30917c3b36c383c28ded0472c35b2/examples/forward_modeling/elastic_modeling2d/forward.sh#L5)

**Comment by zhangxiaoshuotttt** [comment]
3. I will write some codes help convert Sgy format into HDF5, and give some cooments

**Comment by zhangxiaoshuotttt** [comment]
4. [These ](https://github.com/GeophyAI/seistorch/blob/202c77abbdc30917c3b36c383c28ded0472c35b2/examples/check_features/torchrun_dist/inversion.sh#L1C10-L1C28)should be changed

**Comment by zhangxiaoshuotttt** [comment]
5. [This](https://github.com/GeophyAI/seistorch/blob/202c77abbdc30917c3b36c383c28ded0472c35b2/examples/check_features/torchrun_dist/inversion.sh#L6) is useless, It should be `step-per-epoch`

**Comment by GeophyAI** [comment]
Thank you for all your comments on seistorch.
Q1: I've added a alert when the boundary with is not 50.
Q2: I've modified the script to the right one. Additionally, u can refer to `examples/forward_modeling/different_eq`  for more modeling examples.
Q3: There's a segyreader class in `seistorch/io.py`, u can develope that based on my codes.
Q4: The argument` --nproc_per_node `should be set as the number of GPU u have.
Q5: Yes, u r right. All the`--num-batches` in inversion script should be modified to `--step-per-epoch` (I will modify those scripts gradually in the coming updates).
